src/sage_docbuild/ext/inventory_builder.py

```diff
@@ -8,13 +8,15 @@
 
 from os import path
 from pathlib import Path
-from typing import Any, Iterable
+from typing import Any, Iterable, TYPE_CHECKING
 from urllib.parse import quote
 
-from sphinx.application import Sphinx
 from sphinx.builders.dummy import DummyBuilder
 from sphinx.util.inventory import InventoryFile
 
+if TYPE_CHECKING:
+    from sphinx.application import Sphinx
```

Comment:
This file isn't included in the CI run of ruff because it isn't in `src/sage` but I ran the ruff formatter on all of `src/` anyway.